pkg/controller/mizar/mizar-service-controller.go

```diff
@@ -211,208 +212,160 @@ func (c *MizarServiceController) syncService(eventKeyWithType KeyWithEventType)
 	return nil
 }
 
-func (c *MizarServiceController) processServiceCreation(service *v1.Service, eventKeyWithType KeyWithEventType) error {
+func (c *MizarServiceController) processServiceCreateOrUpdate(service *v1.Service, eventKeyWithType KeyWithEventType) error {
 	key := eventKeyWithType.Key
-	netName := getArktosNetworkName(service.Name)
-	networkName, hasDNSServiceLabel := service.Labels[arktosapisv1.NetworkLabel]
+	tenant, _, _, err := cache.SplitMetaTenantNamespaceKey(key)
+	if err != nil {
+		return err
+	}
+
+	// Get tenant default network
+	tenantDefaultNetwork, err := c.networkLister.NetworksWithMultiTenancy(tenant).Get(defaultNetworkName)
+	if tenantDefaultNetwork.Spec.Type != mizarNetworkType {
+		return nil
+	}
+
+	if tenantDefaultNetwork.Status.Phase != arktosapisv1.NetworkReady {
+		return errors.New(fmt.Sprintf("The arktos network %s is not Ready.", tenantDefaultNetwork.Name))
```

Comment:
error msgs, please use lower case for first char.

Reply:
Fixed.